overlord/devicestate/handlers_gadget.go

```diff
@@ -84,6 +84,11 @@ var (
 	gadgetUpdate = gadget.Update
 )
 
+func (m *DeviceManager) setGadgetRestartRequired(t *state.Task) {
```

Comment:
this helper doesn't need to be a DeviceManager method as it's not using m

Reply:
Yea that's a good point, thanks!